### ISSUE TITLE
Upgrade to Ionic 2

### ISSUE THREAD

**Issue by lorensr** [JoeyAndres/meteor-ionic#65]
It's supposed to have a lot of improvements, including a different native-like UI for Android. 

Might be a large undertaking, but could keep this issue to track the idea & who's interested. Currently you have to use Meteor with Angular 2 in order to use Ionic 2.

**Comment by lorensr** [comment]
Looks like the community is moving away from Blaze to React. Especially since it's relatively easy to [mix Blaze and React](http://guide.meteor.com/react.html) for legacy apps, might be nice to combine efforts on a React Ionic 2: 

https://github.com/reactionic/reactionic/issues/2

**Comment by cyclops24** [comment]
@lorensr I agree with you but, I think Blaze still have their own fans. For example I really love Blaze because It's more simpler and for my projects and my start-up scale it work's well so I think It's really depend on project scale and It's a big mistake if all of community move to React without any reason.

It's so good to improve things and switch to new technology but when we need it. Not just for fun.

**Comment by JoeyAndres** [comment]
@lorensr Me and @cyclops24 were thinking of divorcing from blaze. We were thinking of using polymer, considering it is using the webcomponent technology which is currently being standardised in w3c and some of its functionality is already native in chrome. We were thinking that this will be a npm package and meteor package, meaning it will be independent of meteor framework too. This way we maximize audience and future proof ourselves a bit (considering its being standardised in w3c).

**Comment by lorensr** [comment]
@cyclops24 I think Blaze is simpler to use as well (not to mention things like useraccounts and autoform), and it might the best choice for simple things. 

If you wanted to go for maximum usefulness to others, then I think you'd have more people using it (and probably helping develop) if it were done in React than if it were done in Blaze or Polymer. Compare 15k GH stars on polymer vs 40k on React. For comparisons on things other than maximal audience, this looks pretty good: http://programmers.stackexchange.com/a/237762/109611

I agree that whatever it is, it would be better as an npm package.

**Comment by JoeyAndres** [comment]
@lorensr Thanks for the input. Me and @cyclops24 will discuss this again later.

**Comment by JoeyAndres** [comment]
@lorensr Also one more thing. I'll just finish this one (1.2.4 or 1.3) so this will not haunt me later in life. Rest assured me and @cyclops24  are dying to start a meteor agnostic one.
